core/templates/pages/story-editor-page/editor-tab/story-editor.component.html

```diff
@@ -264,6 +264,7 @@ <h3 class="story-card-header">Story Card</h3>
                 <select [(ngModel)]="selectedChapterIndexInPublishUptoDropdown"
                         (ngModelChange)="updatePublishUptoChapterSelection(selectedChapterIndexInPublishUptoDropdown)"
                         class="publish-upto-chapter-dropdown">
+                  <option [value]="-1">0</option>
```

Comment:
This should be "None" if the idea is to publish no chapters, since there is no "chapter 0".

Reply:
Done